content/zh-cn/containers_as_a_service.md

```diff
@@ -6,13 +6,13 @@ category: 技术
 
 ## 是什么
 
-容器即服务（CaaS）是一种云服务，有助于使用基于 [容器](/zh/container/) 的 [抽象](/abstraction/) 管理和部署应用程序。
+容器即服务（CaaS）是一种云服务，有助于使用基于 [容器](/zh-cn/container/) 的 [抽象](/zh-cn/abstraction/) 管理和部署应用程序。
 这项服务可以部署在企业内部或云中。
 
 CaaS 供应商提供了一个框架或协调平台，使容器部署和管理的关键 IT 功能自动化。
-它帮助开发者建立安全和 [可扩展](/scalibility/) 的容器化应用。
+它帮助开发者建立安全和 [可扩展](/zh-cn/scalibility/) 的容器化应用。
```

Comment:
Please fix the typo which makes the link be broken. 😊

`scalibility` → `scalability`